osbs/core.py

```diff
@@ -837,13 +837,14 @@ def import_image(self, name, stream_import):
 
         stream_import['metadata']['name'] = name
         for tag in imagestream_json.get('spec', {}).get('tags', []):
```

Comment:
Maybe convert `tags` to a set to make it a bit more efficient and to be clear of how it's going to be used?

Reply:
ok